salt/modules/openscap.py

```diff
@@ -107,3 +107,7 @@ def xccdf(params):
         upload_dir=upload_dir,
         error=error,
         returncode=returncode)
+
+
+def xccdf_eval(profile=None, policy=None):
+    return xccdf('eval --profile {0} {1}'.format(profile, policy))
```

Comment:
@isbm meanwhile we discussed this. just for the record, it was because we needed a way just pass the oscap parameters as a string to salt.
We might deprecate this in the future if there's no use for it.